tests/network/vm_import/conftest.py

```diff
@@ -0,0 +1,254 @@
+from collections.abc import Generator
+from typing import Final
+
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.forklift_controller import ForkliftController
+from ocp_resources.migration import Migration
+from ocp_resources.namespace import Namespace
+from ocp_resources.network_map import NetworkMap
+from ocp_resources.plan import Plan
+from ocp_resources.provider import Provider
+from ocp_resources.resource import ResourceEditor
+from ocp_resources.secret import Secret
+from ocp_resources.storage_map import StorageMap
+from pytest_testconfig import config as py_config
+
+from libs.net.udn import create_udn_namespace
+from tests.network.libs import cluster_user_defined_network as libcudn
+from tests.network.libs.label_selector import LabelSelector
+from tests.network.vm_import.source_provider import SourceClusterProvider
+from utilities.bitwarden import get_cnv_tests_secret_by_name
+
+CUDN_LABEL: Final[dict] = {"cudn": "mtv"}
+CUDN_SUBNET_IPV4: Final[str] = "192.168.100.0/24"
+MTV_RESOURCE_TIMEOUT_SEC: Final[int] = 30
+VDDK_IMAGE: Final[str] = "quay.io/libvirt_v2v_cnv/vddk:8.0.1"
+
+
+@pytest.fixture(scope="session")
+def source_cluster_data() -> dict:
```

Comment:
nit: just saw this - i would suggest renaming it to `source_hypervisor_data` to reflect the data that is returned (as it is not related to a cluster in this context) and recommend any other fixtrures that use "source_cluster" to be renamed accordingly.